conan/tools/cmake/cmakedeps/templates/config_version.py

```diff
@@ -19,18 +20,28 @@ def filename(self):
 
     @property
     def context(self):
-        return {"version": self.conanfile.ref.version}
+        policy = self.conanfile.cpp_info.get_property("cmake_package_version_compat")
```

Comment:
Just double checking on the name - if the `-version.cmake` and these contents are only generated when the `cmake_find_mode` is `config`, maybe it makes sense to re-use the word config in the property name? the properties that apply to the find modules are called `cmake_module_xxx`, so maybe this could be `cmake_config_version_compat`?

Reply:
Yes, it does. I was using your initial proposal :) , but I already thought that ``config_version`` made more sense. I will rename it.

Reply:
Done